scripts/prepublish

```diff
@@ -4,4 +4,12 @@ set -e
 rm -f README.md
 make
 sed -i '' "s/var version = '.*';/var version = '$VERSION';/" index.js test/index.js
+node --eval '
+  var fs = require("fs");
+  var pkg = require("./package.json");
+  pkg.devDependencies["sanctuary-type-classes"] = process.env.VERSION;
+  fs.writeFileSync("./package.json",
+                   JSON.stringify(pkg, null, 2) + "\n",
+                   {encoding: "utf8"});
+'
```

Comment:
This is great! It may not be necessary, but it's probably a good idea to add `package.json` to the `git add` arguments list.